cache/compressed_secondary_cache.cc

```diff
@@ -147,14 +148,14 @@ std::string CompressedSecondaryCache::GetPrintableOptions() const {
 
 std::shared_ptr<SecondaryCache> NewCompressedSecondaryCache(
     size_t capacity, int num_shard_bits, bool strict_capacity_limit,
-    double high_pri_pool_ratio,
+    double high_pri_pool_ratio, double low_pri_pool_ratio,
```

Comment:
Same comment as before re: adding new parameters to public APIs

Reply:
Done. Thanks.